internal/config/validate.go

```diff
@@ -12,20 +12,23 @@ func (c *Config) validate() error {
 		errs.appendFieldError("BUILDKITE_SPLITTER_RETRY_COUNT", "was %d, must be greater than or equal to 0", c.MaxRetries)
 	}
 
-	if got, min := c.Parallelism, 1; got < min {
+	if c.Parallelism == nil {
+		errs.appendFieldError("BUILDKITE_PARALLEL_JOB_COUNT", "not set in environment")
```

Comment:
this should say `must not be blank` to bring it in line with the other error messages